core/src/Ribasim.jl

```diff
@@ -24,7 +24,7 @@ import TranscodingStreams
 using Accessors: @set
 using Arrow: Arrow, Table
 using CodecZstd: ZstdCompressor
-using ComponentArrays: ComponentVector
+using ComponentArrays: ComponentVector, Axis, parent
```

Comment:
```suggestion
using ComponentArrays: ComponentVector, Axis
```

I'm assuming `ComponentArrays.parent` === `Base.parent`, so you don't need to import it.